draft-ietf-quic-transport.md

```diff
@@ -887,6 +887,13 @@ server discards the incoming packet and responds with a Version Negotiation
 packet ({{packet-version}}).  This includes a list of versions that the server
 will accept.
 
+There is no limit to the size of the Version Negotiation packet, other than the
+number of versions that the server supports and the estimated path MTU.  To
```

Comment:
Since the point of this change is to impose a limit, this first sentence seems contradictory.  I'd just drop it.